docs/downstream/crc.md

```diff
@@ -0,0 +1,34 @@
+# Deployment on CRC
+
+These is how to deploy ArgoCD on [CRC](https://developers.redhat.com/products/codeready-containers/overview).
+
+## Instalation Steps
+
+ * Setup CRC https://developers.redhat.com/products/codeready-containers/overview
```

Comment:
Can you convert all your links to hyperlinks  in the form [ text ] ( link )  it's a lot less cluttered this way imo

Reply:
I do it for internal links. For external links I keep the URL in text. I personally like to see where I am heading when I click links. But it's probably a matter of preference.